connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -864,11 +973,15 @@ uint8_t Gap::getMaxWhitelistSize(void) const
 
 ble_error_t Gap::getWhitelist(whitelist_t &whitelist) const
 {
+    tr_info("Get whitelist");
```

Comment:
Do we need a trace to indicate we are using this getter ? The trace in the error path should be sufficient if it mentions that it fails while trying to get the whitelist. However we do need traces in `initialize_whitelist`.